src/main/java/io/jenkins/plugins/tuleap_oauth/guice/TuleapOAuth2GuiceModule.java

```diff
@@ -21,6 +20,6 @@ protected void configure() {
         bind(IDTokenChecker.class).to(IDTokenCheckerImpl.class);
         bind(UserInfoChecker.class).to(UserInfoCheckerImpl.class);
         bind(TuleapAuthorizationCodeUrlBuilder.class).to(TuleapAuthorizationCodeUrlBuilderImpl.class);
-        bind(OkHttpClient.class).toProvider(OkHttpClientProvider.class).asEagerSingleton();
+        install(new TuleapAuthenticationApiGuiceModule());
```

Comment:
where does this install comes from? Do you have any documentation to point me to ?

Reply:
The solution comes from here => https://stackoverflow.com/questions/7128327/how-to-have-a-guice-module-use-another-guice-module